packages/core/strapi/lib/services/entity-service/types/index.d.ts

```diff
@@ -11,4 +11,13 @@ export interface EntityService {
     entityId: number,
     params?: EntityService.Params.Pick<TContentTypeUID, 'fields' | 'populate'>
   ): Promise<Attribute.GetValues<TContentTypeUID> | null>;
+  delete<TContentTypeUID extends Common.UID.ContentType>(
+    uid: TContentTypeUID,
+    entityId: number,
+    params?: EntityService.Params.Pick<TContentTypeUID, 'fields' | 'populate'>
+  ): Promise<Attribute.GetValues<TContentTypeUID> | null>;
+  create<TContentTypeUID extends Common.UID.ContentType>(
+    uid: TContentTypeUID,
+    params?: EntityService.Params.Pick<TContentTypeUID, 'data' | 'files' | 'fields' | 'populate'>
+  ): Promise<Attribute.GetValues<TContentTypeUID>>;
```

Comment:
In theory`create` can return null if the entity is deleted before it's retrieved again, but I consider that a bug in create() and will address that in a different PR targeted at main.